mvg/analysis_classes.py

```diff
@@ -245,8 +325,8 @@ def to_df(self, file_name=None):
         self.check_status()
         if file_name is not None:
             print(f"Saving {self.feature()} data frame results to", file_name)
-            self.dframe.copy().to_csv(file_name, index=False)
-        return self.dframe
+            self._dframe.copy().to_csv(file_name, index=False)
```

Comment:
I think it's better to make a `save_csv` method rather than having this method do two things

Reply:
OK